app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<section class="users-section">
+  <ul class="users-list">
+  <% @inverse_friendships.reverse.each do |request|%>
+    <% unless confirm?(request.sender) %>
```

Comment:
- [ ] There shoud be absolutely NO logic in views.

Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.

Please move the logic to its corresponding helper

Reply:
fixed creating business logic and partials